doc/getting-started/how-helix-works.md

```diff
@@ -0,0 +1,19 @@
+# How does Helix work?
+
+This page takes a behind-the-scenes look at the overall Helix architecture and describes its main components.
+
+![Helix Architecture Overview](/assets/helix-overview.png)
+
+Helix uses the Fastly CDN to deliver content and deploys Helix-specific _VCL edge code_ to process client requests.
+
+That VCL code is generated by the _hlx publish_ command along with the _Helix Publish Service_ deployed in Adobe I/O Runtime.
+Together, they generate and setup the appropriate Fastly configurations based on the contents
+of the source GitHub repository which provides the content, rendering templates and code as well as the 
```

Comment:
I would split out the GitHub column into its own section with its own heading. The three boxes in the GitHub column work very differently and so I think deserve more than a single sentence to describe them. It's also not clear that:

- static website assets (JS/CSS) come from the `htdocs/` folder of your helix site code repo
- rendering templates and code come from the `src/` folder of your helix site code repo
- markdown content can be anywhere

So splitting that out into its own section, IMHO, gives us space to describe this and helps make it clearer.

Reply:
I think the folder names are too much detail at this point but  I'm adding a paragraph starting with "the content, code and static files can come from one or several GitHub repositories..." to give a hint about what the structure can be - and the details will be found somewhere else, eventually.